python/lsst/ts/observatory/control/maintel/mtcs.py

```
@@ -1075,6 +1075,81 @@ async def move_rotator(self, position: float) -> None:
             component_name="MTRotator",
         )
 
+    async def move_m1m3_hp(self, hp_index: int, step_size: int) -> None:
+        """Move a single M1M3 hardpoint (hp) to a new position
+
+        Parameters
+        ----------
+        hp_index: `int`
+            Hardpoints index (from 0 to 5).
+        """
+        steps = [0] * 6
+        steps[hp_index] = step_size
+
+        self.log.info(f"Moving Hardpoint {hp_index} in {step_size} steps - Start")
+        await self.rem.mtm1m3.cmd_moveHardpointActuators.set_start(
+            steps=steps, timeout=self.long_timeout
+        )
+
+        # Wait a bit
+        await asyncio.sleep(1)
+
+        # Verify the commanded actuator is moving
+        hpActuatorState = await self.rem.mtm1m3.evt_hardpointActuatorState.aget()
+        assert (
```

Please, don't use `assert` for this. If you want to raise an exception do:

```
if hp_actuator_state.motionState[hp_index] != MTM1M3.HardpointActuatorMotionStates.STEPPING:
    raise RuntimeError("Relevant message explaining why you are raising an exception.")
```